CONTRIBUTING.md

```diff
@@ -23,7 +23,57 @@ use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
 
+## KPT Packages
+
+In order to maintain consistency accross kpt packages, we strive to maintain the following standards.
+
+### Generic
+
+1. Google's License text is included at the beginning of each resource file
+1. Avoid blank lines
+1. `metadata.name` include the `project-id` prefix only when K8S uniqueness is required
+1. `spec.resourceID` include the `project-id` prefix only when GCP uniqueness is required
+1. Resource's fields/subfields are ordered as per resource's schema
+1. Resource's reference should use `name` instead of `external` as much as possible
+
+### Comments
+
+1. A comment line is included prior to every resources definition
+1. The resource description field has the same information as the comment line at the beginning of the resource definition
+1. Avoid comments on same line as a field
+1. `kpt-merge` comment should be removed
+1. Include a comment for every setter in `setters.yaml`
+
+### Annotations
+
+1. Include
+
+    - `depends-on` for every resource
+
+1. Remove :
+
+    - `cnrm.cloud.google.com/blueprint: 'kpt-fn'`
+
+### Markdown
+
+1. `readme.md` is generated using the command below. Replace `REPO_PACKAGE_DIR` and `LOCAL_PACKAGE_DIR` accordingly.
+
+    ```shell
+    REPO_URL=https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit.git/solutions/REPO_PACKAGE_DIR
```

Comment:
Should the REPO_PACKAGE_DIR have a dollar sign -  `$REPO_PACKAGE_DIR`

Reply:
will keep it like this for now as the instructions are providing sufficient guidance